src/BenchmarksDriver/Program.cs

```diff
@@ -799,6 +807,7 @@ private static Task WriteJobsToSql(ServerJob serverJob, ClientJob clientJob, str
                         [Hardware] [nvarchar](max) NOT NULL,
                         [OperatingSystem] [nvarchar](max) NOT NULL,
                         [Framework] [nvarchar](max) NOT NULL,
+                        [RuntimeStore] [bit] DEFAULT 1,
```

Comment:
I think this column should be `NOT NULL` rather than `DEFAULT 1`.  It's dangerous to default to either 1 or 0.  When creating the column in the prod DB, you can set a default value to convert all existing rows, then remove the default so new rows must specify.